coffee_lib/src/plugin_manager.rs

```diff
@@ -36,6 +36,9 @@ pub trait PluginManager {
     /// list the remote repositories for the plugin manager.
     async fn list_remotes(&mut self) -> Result<CoffeeRemote, CoffeeError>;
 
+    /// List the plugins available in a remote repository.
+    async fn list_plugins_in_remote_repo(&self, name: &str) -> Result<CoffeeList, CoffeeError>;
```

Comment:
```suggestion
    async fn list_plugins_in_remote_repo(&self, name: &str) -> Result<CoffeeList, CoffeeError>;
```

due that this being a public method I would like to change the name to one a little bit more ergonomic like `remote_plugins` or `get_plugins_in_remote`.

`Remote repo` is kind confusing because the remote is a repository in the case of git